tracetools_launch/launch/example.launch.py

```diff
@@ -14,7 +14,7 @@
 
 """Example launch file for the Trace action."""
 
-from launch import LaunchDescription
+from launch.launch_description import LaunchDescription
```

Comment:
@christophebedard This diverges from the [official launch tutorial](https://docs.ros.org/en/humble/Tutorials/Intermediate/Launch/Creating-Launch-Files.html), which I wouldn't do unless there's a good reason (for example, what if the location of the `LaunchDescription` class changes?).  Is it possible to fix the mypy configuration instead so it doesn't give a warning here?

Besides, do you know why mypy warns here? The code looks absolutely correct to me, and the `launch` package [explicitly wants you to import from it](https://github.com/ros2/launch/blob/rolling/launch/launch/__init__.py)

Reply:
@tobiasblass I assumed it was a bug with mypy at first, since it doesn't seem to consider `__all__`, but I found things that seemed to indicate that this was an intentional change (i.e., changing from using `__all__` to doing direct imports). I'll keep looking for a fix.

Reply:
@tobiasblass I just removed the seemingly-unrelated/unnecessary changes for now.